README.md

```diff
@@ -117,14 +209,10 @@ Classifiers are tacked on the end, e.g. `"foo.bar:blah:1.0:jar:some-classifier"`
  
 ## Limitations
 
-This doesn't assign inter-dependencies between maven artifacts, nor does it hook up annotation
-procesors or do other things you might want.  Future versions may support some of these features,
+This doesn't assign inter-dependencies between maven artifacts Future versions may support this,
```

Comment:
This sentence seems damaged.

Reply:
Should be fixed.  I missed a period, but then realized that it could be worded better.